### PR TITLE
Mr. Fresco: Javascript Restaurant Project Webpage

### PR DESCRIPTION
**Pull request by ferbaco86**
Here in this project, I applied the knowledge learned on DOM manipulation, ES6 Modules and Webpack through reading the material and doing Udacity tutorials on Javascript.

### Project Specifications

- [x] Use webpack

- [x] Remember to include all webpack related packages in devDependencies

- [x] Remember to add scripts which are necessary to build your project in package.json

- [x] Create a src and dist directory

- [x] Create a webpack.config.js

- [x] Set up an HTML skeleton inside of dist/index.html with single <div id="content">

- [x] Create the HTML elements by using JavaScript only, e.g. by appending each new element to div#content once the page is first loaded

- [x] Next, set up your restaurant site to use tabbed browsing to access the Contact and Menu pages

- [x] Put the contents of each ‘tab’ inside of it’s own module

### General Requirements

- [x] Add a descriptive README file to your repository.

- [x] The README must include instructions on how to use the project on your machine.

- [x] Use descriptive commit messages.

- [x] Do not commit console.log() instructions.

### GitFlow:

- [x] Create a development branch and set it as a default branch in your repository.

- [x] Create a feature-branch branch (give it a descriptive name)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,87 @@
...
+
+## Getting Started
+
+To running it locally follow these simple steps.
+
+
+### Setup
+
+First build the project with this command:
+
```
Comment (christianotieno) [inline README.md:41]: ## Additional changes for setup instructions
Please add instructions on how to install the dependencies here. Thank you
```suggestion
npm install
```

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes required ♻️ 
Hello @ferbaco86 👋 COngratulations on the hard work. keep it up.
##
Everything looks great, we just need to work on a small pointed out issue down below 👇 before we close this up

### Christian Otieno (TSE)

File: README.md (hunk 1 above)
Comment (ferbaco86) [reply README.md:41]: You are totally right. Thanks for the suggestion!
Already fixed

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Pull request merged**